pagopa/io-app
# chore(IT Wallet): [SIW-1816] Improve mixpanel tracking

## Short description
This PR updates some mixpanel tracking events and adds two new scroll-to-bottom tracking events

## List of changes proposed in this pull request
- Fixed some mixpanel properties
- Added a new event
## How to test
Try to request a credential, check if the events are correctly sent into mixpanel
